config/middleware.py

```diff
@@ -1,51 +1,41 @@
-""" Middleware Configuration Settings """
+"""Middleware Configuration Settings."""
 
-from masonite.middleware import ResponseMiddleware, SecureHeadersMiddleware
+from masonite.middleware import (MaintenanceModeMiddleware, ResponseMiddleware,
+                                 SecureHeadersMiddleware)
 
 from app.http.middleware.AddAttributeMiddleware import AddAttributeMiddleware
-from app.http.middleware.AuthenticationMiddleware import AuthenticationMiddleware
+from app.http.middleware.AuthenticationMiddleware import \
+    AuthenticationMiddleware
 from app.http.middleware.CsrfMiddleware import CsrfMiddleware
 from app.http.middleware.LoadUserMiddleware import LoadUserMiddleware
 from app.http.middleware.MiddlewareTest import MiddlewareTest
+from app.http.middleware.VerifyEmailMiddleware import VerifyEmailMiddleware
 
-
-"""
-|--------------------------------------------------------------------------
-| HTTP Middleware
-|--------------------------------------------------------------------------
-|
-| HTTP middleware is middleware that will be ran on every request. Middleware
-| is only ran when a HTTP call is successful (a 200 response). This list
-| should contain a simple aggregate of middleware classes.
-|
+"""HTTP Middleware
+HTTP middleware is middleware that will be ran on every request. Middleware
+is only ran when a HTTP call is successful (a 200 response). This list
+should contain a simple aggregate of middleware classes.
 """
 
 HTTP_MIDDLEWARE = [
     LoadUserMiddleware,
     # todo
     # CsrfMiddleware,
```

Comment:
@josephmancuso FYI. I have kept it as todo.

Reply:
to uncomment it?

Reply:
I mean I kept it as it is. it commented out when I doing spring cleaning.